main.go

```diff
@@ -262,6 +263,12 @@ func hashBang(b []byte) []byte {
 	return nil
 }
 
+var reGenerated = regexp.MustCompile(`(?m)^// Code generated .* DO NOT EDIT\.$`)
```

Comment:
Including language specific comment syntax ("//") limits scope of support. Would you consider adding other common comment indicators, e.g. '#', '/*', etc.?

Reply:
That's a good idea, What about replacing "//" with ".{0,4}" or something like that? (keeping the space before Code. It could also be ".*".

Reply:
Chose .{1,2}

Reply:
Sorry for the slow response. My concern is that could wipe out lines unintentionally because of the open ended pattern match. I know it's unlikely because of the need to match the "Code generated" part but how about, to be safer, a pattern like this: "^(\/\/ | \/\* | # | rem )" plus any other comment conventions can think of. I know this may seem like overkill but I want to make sure we never silently comment out actual code.

Reply:
What do you mean by wipe out? This regexp is only looked at a marker to determine if the file should be skipped, if so the file is not touched. It doesn't imply anything about the file modification.